vsphere/metadata.csv

```diff
@@ -351,6 +351,10 @@ vsphere.virtualDisk.writeLatencyUS.latest,gauge,,microsecond,,"Write latency in
 vsphere.virtualDisk.writeLoadMetric.latest,gauge,,,,Storage DRS virtual disk metric for the write workload model,0,vsphere,virtualdisk writeLoadMetric latest
 vsphere.virtualDisk.writeOIO.latest,gauge,,request,,Average number of outstanding write requests to the virtual disk,0,vsphere,virtualdisk writeOIO latest
 vsphere.vm.count,gauge,,,,The total number of VMs,0,vsphere,vm count
+vsphere.host.count,gauge,,,,The total number of ESXi Hosts,0,vsphere,host count
+vsphere.datastore.count,gauge,,,,The total number of Datastores,0,vsphere,datastore count
+vsphere.datacenter.count,gauge,,,,The total number of Datacenters,0,vsphere,datacenter count
+vsphere.cluster.count,gauge,,,,The total number of Clusters,0,vsphere,cluster count
```

Comment:
It's maybe better to state something like `Number of XXX tagged by xx, xx, xx`, or just `Number of XXX`.

Total is IMO not suitable for this case, since it's not always total since we can group by tags.

Reply:
👍  Added more details and some explanation about how to use it